src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -104,33 +103,26 @@
     private Set<Vector3i> detectedBlocks = new HashSet<>();
 
     /**
-    * This set the local player for the tests
+    * This sets the local player for the tests
```

Comment:
Usually, `This sets...` is useless. Since this javadoc belongs to this method, it's obvious you're talking about this method. So, repharse this to `Sets the local player..`.